### PR TITLE
Add last scan date below upsell button

### PR DESCRIPTION
**Pull request by Vinnl**
# References: 
Jira: MNTOR-2701
Figma: https://www.figma.com/file/DTbmXzCQCw2PxXpHQW8yfW/Monitor-MVP-Enhancements?type=design&node-id=24-538&mode=design

# Description

Lets a user know when their last scan took place.

Note that I also moved `UpsellBadge.tsx` into the `toolbar` folder, since it's shown in the toolbar.

# Screenshot (if applicable)

image

image

# How to test

Open the dashboard with a user who has performed a scan.

# Checklist (Definition of Done)
- [x] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug. - No real business logic added.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [x] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

**Comment by Vinnl** [comment]
Putting back in draft mode - it needs to be put behind a feature flag.

File: src/db/tables/onerep_scans.ts [hunk 1]
```diff
@@ -40,21 +40,26 @@ async function getScanResults(
...
-    };
+    return null;
   }
 
   const scan = await knex("onerep_scans")
     .first()
     .where("onerep_profile_id", onerepProfileId)
     .orderBy("created_at", "desc");
 
+  return scan ?? null;
```
Comment (pdehaan) [inline src/db/tables/onerep_scans.ts:55]: Interesting! What does this do?
I thought the nullish coalescing would only return the right-hand side if the left-hand side was already null.

Comment (Vinnl) [reply src/db/tables/onerep_scans.ts:55]: Nullish coalescing returns the right-hand side if the left-hand side is `null` _or `undefined`_. So this ensures we return `null` if there was no result.

**Comment by Vinnl** [comment]
@rhelmer I was going to do that as part of setting up the experiment in the first place, in a separate PR, since that has its own ticket (and is probably a fair bit of (review) work by itself).

**Review by rhelmer — APPROVED** [review]

**Comment by Vinnl** [comment]
> LGTM but not sure if we're still rolling this out since I think we're looking to redesign it?

I'm not actually sure if we made a decision on that, but we're going to run an experiment for it, so we'll first have to set that up anyway. Possibly we'll move it over a bit after that.

**Pull request merged**